### PR TITLE
Clean up Adjust Target Leverage

### PR DESCRIPTION
**Pull request by ruixhuang**
* Adding token logo
* Wiring up with Abacus data
* General refactoring
* 
Screenshot_1717780956

### CODE REVIEW

File: v4/feature/portfolio/src/main/java/exchange/dydx/trading/feature/portfolio/components/positions/DydxPortfolioPositionItemView.kt [hunk 1]
```diff
@@ -355,27 +356,28 @@ object DydxPortfolioPositionItemView {
...
-                shape = RoundedCornerShape(4.dp),
-                backgroundColor = ThemeColor.SemanticColor.layer_6,
-                borderColor = ThemeColor.SemanticColor.layer_7,
-            ) {
-                Icon(
-                    painter = painterResource(id = exchange.dydx.trading.common.R.drawable.ic_edit),
-                    contentDescription = "",
-                    tint = ThemeColor.SemanticColor.text_primary.color,
-                )
+            if (position.childSubaccountNumber != null) {
```
Comment (mike-dydx) [inline v4/feature/portfolio/src/main/java/exchange/dydx/trading/feature/portfolio/components/positions/DydxPortfolioPositionItemView.kt]: can do `position.marginMode == isolated` for clarity. Also. I'm not sure but does subaccount 0 have childSubaccountNumber == 0? Guessing not, but haven't seen in abacus.

File: v4/feature/trade/src/main/java/exchange/dydx/trading/feature/trade/targetleverage/DydxTradeInputTargetLeverageView.kt [hunk 2]
```diff
@@ -55,14 +56,20 @@ object DydxTradeInputTargetLeverageView : DydxComponent {
...
             val preview = ViewState(
                 localizer = MockLocalizer(),
                 leverageText = "1.0",
-                leverageOptions = listOf(),
+                leverageOptions = listOf(
+                    LeverageTextAndValue("1.0", "1.0"),
+                    LeverageTextAndValue("2.0", "2.0"),
+                    LeverageTextAndValue("5.0", "5.0"),
+                    LeverageTextAndValue("10.0", "10.0"),
+                ),
```
Comment (mike-dydx) [inline v4/feature/trade/src/main/java/exchange/dydx/trading/feature/trade/targetleverage/DydxTradeInputTargetLeverageView.kt:72]: i think max leverage can sometimes be 5? This is what I did in iOS 

https://github.com/dydxprotocol/v4-native-ios/blob/e89aaba312de14dd0f17f1ab9f9e26eaa46d20ad/dydx/dydxPresenters/dydxPresenters/_v4/Trade/Margin/dydxTargetLeverageViewBuilder.swift#L98

maybe overkill 🤷

**Review by mike-dydx — APPROVED** [review]

File: v4/feature/trade/src/main/java/exchange/dydx/trading/feature/trade/targetleverage/DydxTradeInputTargetLeverageView.kt (hunk 2 above)
Comment (ruixhuang) [reply v4/feature/trade/src/main/java/exchange/dydx/trading/feature/trade/targetleverage/DydxTradeInputTargetLeverageView.kt:72]: This is just for UI preview during development.

File: v4/feature/portfolio/src/main/java/exchange/dydx/trading/feature/portfolio/components/positions/DydxPortfolioPositionItemView.kt (hunk 1 above)
Comment (ruixhuang) [reply v4/feature/portfolio/src/main/java/exchange/dydx/trading/feature/portfolio/components/positions/DydxPortfolioPositionItemView.kt]: Updated

**Review by mike-dydx — APPROVED** [review]

**Pull request merged**